MediaPushPlugin/src/main/resources/e4e4da7454e7cf2e6ffa.js

```diff
@@ -0,0 +1 @@
+var FRAME_PER_SECOND=10,FRAME_INTERVAL=1/FRAME_PER_SECOND;class VolumeMeter extends AudioWorkletProcessor{constructor(){super(),this._lastUpdate=currentTime,this._volume=0,this.stop=!1,this.port.onmessage=e=>{"stop"===e.data&&(this.port.postMessage({type:"debug",message:"Stop command is received"}),this.stop=!0)}}calculateRMS(e){for(var t=0,s=0;s<e.length;s++)t+=e[s]*e[s];var r=Math.sqrt(t/e.length);this._volume=r}process(e,t){var s=e[0][0];return currentTime-this._lastUpdate>FRAME_INTERVAL&&(this.calculateRMS(s),this.port.postMessage(this._volume),this._lastUpdate=currentTime),!this.stop}debug(e){this.port.postMessage({type:"debug",message:e})}}registerProcessor("volume-meter",VolumeMeter);
```

Comment:
I think we can remove these files from repo becuase it's created on build time automatically